configs/vision/owkin/phikon/offline/camelyon16.yaml

```diff
@@ -0,0 +1,130 @@
+---
+trainer:
+  class_path: eva.Trainer
+  init_args:
+    n_runs: &N_RUNS ${oc.env:N_RUNS, 5}
+    default_root_dir: &OUTPUT_ROOT ${oc.env:OUTPUT_ROOT, logs/owkin/phikon/offline/camelyon16}
```

Comment:
I think here we should do `&OUTPUT_ROOT ${oc.env:OUTPUT_ROOT, logs}/owkin/phikon/offline/camelyon16`

Reply:
i thought it would be nice to give the user full control over the log dir. E.g. if you run many experiments, you might want to indicate some used settings in the folder name which makes analysis more convenient.

Reply:
that also make sense, but i though that since we have the output hash logger, it will create a unique folder either-way right? The main reason though is that if you want to run multiple experiments and save it in another destination (say `eva-logs/` instead of 'logs/') you have to reset for every config that you are doing the new path -- in the otherway, you just set the `OUTPUT_ROOT` once

however, I see that in the other configs we do the same as here, so i'm proposing lets keep it like this and if we think the other approach is better, we update all of them